integration/ig/non-k8s/trace_dns_test.go

```diff
@@ -152,7 +149,7 @@ func TestTraceDns(t *testing.T) {
 	}
 
 	dnsCmds := []string{
-		"/dnstester & sleep 2", // wait to ensure dns server is running
```

Comment:
It seems we were suppose to use `&&` here so I fixed it, sharing to make sure it is caught in the review!

Reply:
Ops, yes you are right. Not sure what was I thinking. :laughing: I will revert the change!